README.md

```diff
@@ -117,6 +117,15 @@ Success! Data written to: development/webapp/cockpit_api_pass
 
 Note: if secret value is equal to existing one, write is skipped. `--force` flag can be used to overwrite.
 
+### Write file content
```

Comment:
Missing information about how to handle gziped base64 encoded a secret file.

Reply:
added: https://github.com/AirHelp/treasury/pull/88/commits/cb3669b31fa883d7f325bb3eb72f25e56f7f9e0e